rest-service/manager_rest/resource_manager.py

```diff
@@ -722,10 +722,6 @@ def validate_plugin_is_installed(self, plugin):
         if not - raises an appropriate exception.
         :param plugin: A plugin from the blueprint
         """
-        # if the plugin is 'diamond' we don't want it to be installed
```

Comment:
I agree that this needs to happen, but probably not in this PR

Reply:
I don't know how this change made it here.
It belongs in another JIRA that I'm sure I had already merged. I'll check.

Reply:
@mcouthon looks like this hasn't been approved/merged yet. Can you approve? https://github.com/cloudify-cosmo/cloudify-manager/pull/1064